lib/scumblr_tasks/security/curl_analyzer.rb

```diff
@@ -70,7 +70,7 @@ def self.options
                           required: false,
                           type: :string},
       :key_suffix => {name: "Key Suffix",
-                      description: "Provide a key suffix for testing out experimental regularz expressions",
+                      description: "Provide a key suffix for testing out experimental regular expressions",
```

Comment:
I probably skipped this because in some languages `z` is meaningful suffix and thus there was no reason for me to spend time investigating it.